### PR TITLE
Create ari.md

### PR DESCRIPTION
**Pull request by arilivigni**
Adds Ari as a participant in the workshop
cc: @mfilosa

### CODE REVIEW

File: Participants/ari.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Hi, I'm [your-github-username]
```
Comment (nathos) [inline Participants/ari.md]: Hey @arilivigni , could you replace `[your-github-username]` with your name please?

**Review by nathos — CHANGES_REQUESTED** [review]

File: Participants/ari.md (hunk 1 above)
Comment (arilivigni) [reply Participants/ari.md]: I changed it from Ari to arilivigni which is my github username

**Review by arilivigni — COMMENTED** [review]
Sorry fixed the Hi line @nathos

**Comment by arilivigni** [comment]
@i-marsh can you review after my changes thanks!

**Review by nathos — APPROVED** [review]
@arilivigni Perfect! This looks great 👍. Go ahead and merge then delete your branch.

emmastone_thumbsup

**Review by i-marsh — APPROVED** [review]
Looks good ... merge and then delete your branch 👍 
:octocat:

**Pull request merged**